### DISCUSSION TITLE
Camera quantity limits

### DISCUSSION THREAD

**Discussion by rugolini** [blakeblackshear/frigate#7235] [General]
Hi there. I'm using Frigate at home and its been awesome.
Now my condo asked me to implement Frigate on the condo system, :-D
Questions:
1. Is there a hard limit to the number of cameras Frigate support?
2. In the case I have a powerfull machine (intel cpu, lots of ram, etc) what would be the camera quantity limit with NO detection, just recording?
Thanks in advance

**Comment by NickM-27** [comment]
1. No, just whatever your system CPU, GPU, and detector can handle
2. Frigate still decodes the stream so you would definitely want to use the GPU for decoding, but recording is very lightweight so the limiting factor would probably be how fast the storage is
